source/chapters/user_interface.rst

```diff
@@ -707,38 +707,123 @@ Channel Faders and Level Meters
+  range. If the Kill switches do not work as expected, check the high/low shelf
+  EQ settings in the preferences.
+
+.. seealso:: You can customize the EQ settings in
+             :menuselection:`Preferences --> Equalizer`.
+
+.. _interface-pfl:
+
+PFL/Headphone Button
+--------------------
+
+.. figure:: ../_static/2.2/headphone-button.png
+   :align: center
+   :width: 321px
+   :figwidth: 100%
+   :alt: The headphone buttons of both decks in the mixer
+   :figclass: pretty-figures
+
+   The headphone buttons of both decks in the mixer
+
+The headphone button is also known as the :term:`pre-fader listen or PFL <PFL>`
+button and allows you to use headphone cueing. That is, it allows you to hear
+the track you will play next in your headphones before your audience hears it.
+This helps you plan when to start the next track and manipulate it to
+make it blend better with the currently playing track (for example, by
+synchronizing the beats, setting a loop, or using EQs). Pressing the PFL button
+plays the track on the headphone output at full volume regardless of the
+position of the crossfader or the deck’s line fader. To play a track in your
+headphones but not have the audience hear it, press the PFL button and move the
+crossfader all the way to the opposite side of the deck or turn the deck’s
+volume fader all the way down.
+
+.. note:: Headphone cueing is only available if you have configured a
+          Headphone Output in :menuselection:`Preferences --> Sound Hardware`.
+
+.. note:: If you use an external hardware mixer, plug your headphones into that
+          rather than your sound card. Use the headphone buttons on the mixer
+          rather than the buttons in Mixxx.
+
+.. _interface-crossfader:
+
+Crossfader
+----------
+
+.. figure:: ../_static/2.2/crossfader.png
+   :align: center
+   :width: 318px
+   :figwidth: 100%
+   :alt: The crossfader section of the mixer
+   :figclass: pretty-figures
+
+   The crossfader section of the mixer
+
+The :term:`crossfader` determines the actual volume of each deck when moving
+the slider from left to right. If both decks are playing and the crossfader is
+in its default center position, then you will hear both decks. Right-clicking on
+the crossfader will reset the slider to its default position.
+
+To move the crossfader left or right, you can use the :term:`keyboard shortcut <shortcut>` :kbd:`G` and :kbd:`H`.
+To apply small crossfade left and small crossfade right, the :term:`shortcuts <shortcut>` are :kbd:`Shift` + :kbd:`G` and :kbd:`Shift` + :kbd:`H` respectively.
+
+You can reverse the configuration of the crossfader, so that the right deck is
+on the left end of the crossfader and vice versa. This is also known as
+*Hamster Style*. To adjust the crossfader to your style of mixing, go to
+:menuselection:`Preferences --> Crossfader`.
+
+.. hint :: Using the :ref:`AutoDJ <djing-auto-dj>` feature in Mixxx, you can
+           automate the crossfade between the decks.
```

Comment:
Below we need an image of the Deere Master/Head/Booth controls.
In Deere, those are located in the tool bar.
You can use this image and store/link it as ../_static/2.2/master-head-controls.png
![image](https://user-images.githubusercontent.com/5934199/98445072-e3836100-2115-11eb-95f7-d58245fd1805.png)

This requires to describe the controls in the order they appear in the GUI.

Reply:
I have added this Image

Reply:
Yes, but it should replace the old below the header:
![image](https://user-images.githubusercontent.com/5934199/99548543-0c7ae000-29b9-11eb-821b-659e0d325fb7.png)